docs/website/content/organizations/turall.md

```diff
@@ -0,0 +1,4 @@
+---
+link: https://github.com/Turall
+title: Tural
```

Comment:
Turall ? :)

Reply:
Hmm, on https://github.com/Turall it's Tural.

Reply:
but perhaps best to use the username

Reply:
but on https://github.com/open-policy-agent/opa/pull/6811/files#diff-45f53c4dd58caae0eef2f5e763d68be001efd052297d3bc7692e7692c4271017R3 I have used the GH account name.

Reply:
Heh, either is fine